JourneysSquad/JourneysSquad/View/AddBillView/DataInputFieldViewAlignment.swift

```diff
@@ -0,0 +1,39 @@
+import SwiftUI
+
+/// An alignment position of the elements inside `DataInputFieldView` along the horizontal axis.
+///
+/// Use to align views of the label and text field inside the root view.
+enum DataInputFieldViewAlignment {
+    /// Aligns the leading edges of views.
+    case leading
+
+    /// Aligns the centres of views.
+    case center
+
+    /// Aligns the trailing edges of views.
+    case trailing
+
+    /// The corresponding value of the `HorizontalAlignment`.
```

Comment:
Correspond for what? :)

Reply:
Corresponding to the case.

Reply:
It seems as non-understandable for me. What about suggestion below ?

```suggestion
    /// Value of `HorizontalAlignment` based on the alignment case.
```

Reply:
Fixed